### PR TITLE
chore: update ursa docker compose to latest grafana and expose all ports

### PR DESCRIPTION
**Pull request by ozwaldorf**
## What

fixes for `infra/ursa/docker-compose.yml`:

- update grafana version
- expose proper ports and update comments

### CODE REVIEW

**Review by b0xtch — DISMISSED** [review]

File: infra/ursa/docker-compose.yml [hunk 1]
```diff
@@ -75,13 +75,12 @@ services:
       dockerfile: Dockerfile
     restart: on-failure
     ports:
-      # 6009 TCP and UDP, used by the P2P protocol running the network
       - "6009:6009"
+      - "4069:4069"
```
Comment (b0xtch) [inline infra/ursa/docker-compose.yml:79]: 4069 doesn’t need to be exposed, as we use nginx in front

Comment (ozwaldorf) [reply infra/ursa/docker-compose.yml:79]: Hmm this actually opens an interesting problem: nodes currently register with the tracker with their raw internal port, there's no consideration for the proxy or https atm. I think this works temp until #151 is completed, which would make the necessary changes here. Also, the prometheus container is scraping ursa:4069, so it needs to be exposed for that internally.

Comment (b0xtch) [reply infra/ursa/docker-compose.yml:79]: We can draft then until that's ready

Comment (b0xtch) [reply infra/ursa/docker-compose.yml:79]: Nodes should register their domain ie tls

Comment (b0xtch) [reply infra/ursa/docker-compose.yml:79]: Using ports exposes on the Host you can scrape with no problem

Comment (ozwaldorf) [reply infra/ursa/docker-compose.yml:79]: > Using ports exposes on the Host you can scrape with no problem

Ah this is the missing link, perfect. So I can just remove the ports and leave expose

**Review by b0xtch — APPROVED** [review]

File: infra/ursa/docker-compose.yml (hunk 1 above)
Comment (ozwaldorf) [reply infra/ursa/docker-compose.yml]: https://github.com/fleek-network/ursa/pull/163/commits/283935721376edbf60cbb18478de76f7325c19b8

**Pull request merged**